File: MonoEngineV2/Vector2f.cpp
#include "pch.h"
#include "Vector2f.h"

Vector2f::Vector2f(_In_ float x, _In_ float y)
	: x(x), y(y)
{ }

Vector2f::Vector2f()
	: Vector2f(0.0f, 0.0f)
{ }

File: MonoEngineV2Lib/Vector2f.cs
using System.Runtime.InteropServices;

namespace MonoEngineV2Lib
{
    [StructLayout(LayoutKind.Sequential)]
    public struct Vector2f
    {
        public float x;
        public float y;

        public Vector2f(float _x, float _y)
        {
            x = _x;
            y = _y;
        }
    }
}